src/modules/token/fungible/components/BalanceInfo/ActionBar.test.js

```diff
@@ -50,27 +49,6 @@ describe('Reclaim balance screen', () => {
     expect(html).not.toContain('open-add-vote-dialog');
   });
 
-  it('Should not display register delegate button', () => {
```

Comment:
please revert back the test and only remove BTC logic

Reply:
Yeah you're right. I'll revert it